code/modules/food_and_drinks/food/snacks_pastry.dm

```diff
@@ -35,72 +38,291 @@
+	name = "apple jelly donut"
+	desc = "Goes great with a shot of cinnamon schnapps."
+	icon_state = "jelly_green"
+	bonus_reagents = list("applejuice" = 3, "sprinkles" = 1, "vitamin" = 1)
+	tastes = list("jelly" = 1, "donut" = 3, "green apples" = 1)
+	is_decorated = TRUE
+	filling_color = "#6ABE30"
+
+/obj/item/reagent_containers/food/snacks/donut/jelly/caramel
+	name = "caramel jelly donut"
+	desc = "Goes great with a mug of hot coco."
+	icon_state = "jelly_beige"
+	bonus_reagents = list("caramel" = 3, "sprinkles" = 1, "vitamin" = 1)
+	tastes = list("jelly" = 1, "donut" = 3, "buttery sweetness" = 1)
+	is_decorated = TRUE
+	filling_color = "#D4AD5B"
+
+/obj/item/reagent_containers/food/snacks/donut/jelly/choco
+	name = "chocolate jelly donut"
+	desc = "Goes great with a glass of warm milk."
+	icon_state = "jelly_choc"
+	bonus_reagents = list("hot_coco" = 3, "sprinkles" = 1, "vitamin" = 1) //the coco reagent is just bitter.
+	tastes = list("jelly" = 1, "donut" = 4, "bitterness" = 1)
+	decorated_icon = "jelly_choc_sprinkles"
+	filling_color = "#4F230D"
+
+/obj/item/reagent_containers/food/snacks/donut/jelly/blumpkin
+	name = "blumpkin jelly donut"
+	desc = "Goes great with a mug of soothing drunken blumpkin."
+	icon_state = "jelly_blue"
+	bonus_reagents = list("blumpkinjuice" = 3, "sprinkles" = 1, "vitamin" = 1)
+	tastes = list("jelly" = 1, "donut" = 2, "blumpkin" = 1)
+	is_decorated = TRUE
+	filling_color = "#2788C4"
+
+/obj/item/reagent_containers/food/snacks/donut/jelly/bungo
+	name = "bungo jelly donut"
+	desc = "Goes great with a mason jar of hippie's delight."
+	icon_state = "jelly_yellow"
+	bonus_reagents = list("bungojuice" = 3, "sprinkles" = 1, "vitamin" = 1)
+	tastes = list("jelly" = 1, "donut" = 3, "tropical sweetness" = 1)
+	is_decorated = TRUE
+	filling_color = "#DEC128"
+
+/obj/item/reagent_containers/food/snacks/donut/jelly/matcha
+	name = "matcha jelly donut"
+	desc = "Goes great with a cup of tea."
+	icon_state = "jelly_olive"
+	bonus_reagents = list("teapowder" = 3, "sprinkles" = 1, "vitamin" = 1)
+	tastes = list("jelly" = 1, "donut" = 3, "matcha" = 1)
+	is_decorated = TRUE
+	filling_color = "#879630"
+
+//////////////////////////SLIME DONUTS/////////////////////////
+
 /obj/item/reagent_containers/food/snacks/donut/jelly/slimejelly
 	name = "jelly donut"
 	desc = "You jelly?"
-	icon_state = "jdonut1"
+	icon_state = "jelly"
 	extra_reagent = "slimejelly"
-	foodtype = JUNKFOOD | GRAIN | FRIED | TOXIC | SUGAR
-
-/obj/item/reagent_containers/food/snacks/donut/jelly/cherryjelly
-	name = "jelly donut"
-	desc = "You jelly?"
-	icon_state = "jdonut1"
-	extra_reagent = "cherryjelly"
-	foodtype = JUNKFOOD | GRAIN | FRIED | FRUIT
-
-/obj/item/reagent_containers/food/snacks/donut/meat
-	bonus_reagents = list("ketchup" = 1)
-	list_reagents = list("nutriment" = 3, "ketchup" = 2)
-	tastes = list("meat" = 1)
-	foodtype = JUNKFOOD | MEAT | GROSS | FRIED
-
-/obj/item/reagent_containers/food/snacks/donut/semen
```

Comment:
:+1:
If someone ever wants to readd them, I guess they'll also have to bring their sprites up to date.

Reply:
SHHH